airflow/models.py

```diff
@@ -4001,27 +4025,42 @@ def get_one(
 
         query = (
             session.query(cls.value)
-            .filter(and_(*filters))
-            .order_by(cls.execution_date.desc(), cls.timestamp.desc())
-            .limit(1))
+                .filter(and_(*filters))
+                .order_by(cls.execution_date.desc(), cls.timestamp.desc())
+                .limit(1))
```

Comment:
This is in the same function. Let's just change it since it is bad code right now and it will likely never get fixed otherwise.